### PR TITLE
true layerwise for myopia

### PR DESCRIPTION
**Pull request by Gurkenglas**
`import torchviz; torchviz.make_dot(torch.stack(loss).sum()).render(filename='lossdetach',format='png')`

lossdetach

### CODE REVIEW

**Comment by neverix** [comment]
lftm
